app/web/src/components/IdeViewer/IdeViewer.js

```diff
@@ -9,33 +9,16 @@ import {
 } from 'react-three-fiber'
 import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls'
 import { Vector3 } from 'three'
-import { STLLoader } from 'three/examples/jsm/loaders/STLLoader'
 import { requestRender } from 'src/helpers/hooks/useIdeState'
 
 extend({ OrbitControls })
 
-function Asset({ url, resetLoading, setLoading }) {
-  const [loadedGeometry, setLoadedGeometry] = useState()
+function Asset({ geometry: incomingGeo }) {
```

Comment:
Asset component is much cleaner now as we're making the three geometry higher in the stack.